pyproject.toml

```
@@ -229,3 +229,6 @@ conflicts = [
 
 [tool.ruff]
 lint.extend-select = ["I"]
+
+[too.ruff.per-file-target-version]
```

```suggestion
[tool.ruff.per-file-target-version]
```
Should help understand how to format/lint that 1 file better.